### ISSUE TITLE
Bug: The furnace doesn't work

### ISSUE THREAD

**Issue by Vincent14** [tsunamayo/Starship-EVO#5055]
### Short Description of the issue:

I sent ores into the furnace and it doesn't produce bars or ingots

### Steps to reproduce the issue:

[Video](https://youtu.be/nRuTjeHR7S0)

### Environment
- Starship EVO version: 22w50c
- Graphics Card: Geforce 2070 RTX

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by ProPeach** [comment]
Was there a reactor on the grid for this one? The furnace probably needs power to work

**Comment by Vincent14** [comment]
No reactor on the grid, it wasn't visible on the footage video I tried to reproduce

**Comment by ProPeach** [comment]
Ah, try putting a reactor down then. It'd be nice if there was a visual aid to show the furnace is on an powered correctly, like the glow only appearing when the grid is powered

**Comment by Vincent14** [comment]
Good catch, I updated my issue in consequence, thanks

**Comment by ProPeach** [comment]
Nice one. To be honest that behaviour could be extended to all systems. Thrusters only glowing when powered and providing thrust, reactors only glowing when fueled and producing power etc. It'd make things a lot clearer and intuitive

**Comment by tsunamayo** [comment]
yes I agree all system should show the deactivated color when power is missing and they dont work. Also the furnace model is still wip, I am waiting to see how it will fit in survival before I spend time on the texture.

**Comment by Vincent14** [comment]
Thanks a lot to have it in minds!
